db/migrate/20230712140334_add_fields_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddFieldsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    # add_column :users, :name, :string
+    # add_column :users, :photo, :string
+    # add_column :users, :bio, :text
+    # add_column :users, :posts_counter, :integer
+  end
```

Comment:
- [ ] I observed you newly added commented-out codes after the last review, kindly ensure that the migration file is not empty for smooth running of the database. 👍